### DISCUSSION TITLE
How to remove duplicates

### DISCUSSION THREAD

**Discussion by ariel-frischer** [saulpw/visidata#1478] [Q&A]
Is there any easy way to just remove all duplicate rows - depending on the current column I am on?

**Comment by saulpw** [comment]
Hi @ariel-frischer, you will probably be interested in @jsvine's dedupe plugin: https://github.com/jsvine/visidata-plugins/blob/stable/plugins/dedupe.py

**Reply by ariel-frischer** [reply]
Thank you! Just tried it works well!
